doc/contribute/contributor_expectations.rst

```diff
@@ -11,14 +11,46 @@ benefits:
   to set aside a few minutes to review smaller changes several times than it is
   to allocate large blocks of time to review a large PR.
 
-- Less wasted work if reviewers or maintainers reject the direction of the
-  change.
+- Contentious and long discussions have a more restricted "splash damage"
+  because they cannot drown unrelated topics that have successfully escaped
+  in separate Pull Requests.
```

Comment:
I would rephrase this so that it is simpler to understand, perhaps:

- The risk of the PR being derailed or side tracked by unrelated topics becomes significantly smaller.